Content.Client/Cargo/UI/CargoConsoleMenu.xaml.cs

```diff
@@ -146,6 +148,15 @@ public void PopulateOrders()
 
             foreach (var order in Owner.Orders.Orders)
             {
+                var productName = Owner.Market.GetProduct(order.ProductId)?.Name;
+
+                if (productName == null)
+                {
+                    DebugTools.Assert(false);
+                    Logger.ErrorS("cargo", $"Unable to find product name for {order.ProductId}");
+                    continue;
+                }
+
```

Comment:
undocumented?

Reply:
I got this exception once and could never replicate it after I added this.